.github/workflows/acceptance.yml

```
@@ -23,6 +23,17 @@ jobs:
       VISUAL_HTML_ENABLE: 1
     steps:
       - uses: actions/checkout@v2
+        name: Checkout sentry (pull_request)
```

```suggestion
        name: Checkout sentry (pull_request_target)
```

Let's just elaborate here that it's for pull_request_target. Also same change in the other place.